src/applications/find-va-forms/components/CardRow.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classnames from 'classnames';
+
+export default function CardRow({ children: cards }) {
+  const columnClassName = classnames(
+    'vads-l-col--12',
+    'vads-u-margin-bottom--1',
+    'vads-u-margin-right--1',
+    'medium-screen:vads-l-col',
+  );
+
+  return (
+    <div className="vads-l-grid-container vads-u-padding--0">
+      <div className="vads-l-row">
+        {cards.map((card, index) => (
+          <div key={index} className={columnClassName}>
+            {card}
+          </div>
```

Comment:
I actually didn't know you could iterate across `children` until I tried it. Is that a new React thing?

Reply:
I don't _believe_ this is new, but I do remember that the convention used to be using `React.Children.map` a few years ago, but this looks way cleaner tbh! 💯

Reply:
Ah, I was thinking we still had to that funky thing where you clone thing child components in order to iterate!

Reply:
Ya I hated that experience haha

Reply:
Yeah me too. Felt so wasteful!